README.md

```diff
@@ -56,9 +56,14 @@ There are three primary controls to determine the inclusion of a device within o
 
 > The plugin is compatible with Nautobot 1.0.0 and higher
 
-The plugin relies on `nautobot_plugin_nornir` to be installed and both plugins to be enabled in your `configuration.py` settings.
+The plugin relies on `nautobot_plugin_nornir` to be installed and both plugins to be enabled in your configuration settings.
 
 ```python
+def is_truthy(arg):
+    if isinstance(arg, bool):
+        return arg
+    return bool(strtobool(arg))
```

Comment:
Do you also need to add the appropriate import for `strtobool`?

Reply:
eek, thanks :)

Reply:
updated